### PR TITLE
feat: Add Dispatcher to casbin v2

### PR DESCRIPTION
**Pull request by dovics**
Signed-off-by: dovics <wrs369@163.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@nodece

File: enforcer.go [hunk 1]
```diff
@@ -338,6 +351,11 @@ func (e *Enforcer) EnableAutoNotifyWatcher(enable bool) {
 	e.autoNotifyWatcher = enable
 }
 
+// EnableautoNotifyDispatcher controls whether to save a policy rule automatically notify the Dispatcher when it is added or removed.
+func (e *Enforcer) EnableautoNotifyDispatcher(enable bool) {
```
Comment (nodece) [inline enforcer.go:355]: modify to `EnableAutoNotifyDispatcher`, same question on beta branch.

File: enforcer_interface.go [hunk 2]
```diff
@@ -51,6 +52,7 @@ type IEnforcer interface {
 	EnableEnforce(enable bool)
 	EnableLog(enable bool)
 	EnableAutoNotifyWatcher(enable bool)
+	EnableautoNotifyDispatcher(enable bool)
```
Comment (nodece) [inline enforcer_interface.go:55]: modify to `EnableAutoNotifyDispatcher`, same question on beta branch.

Comment (dovics) [reply enforcer_interface.go]: I will fix this typo after this PR is merged

**Comment by nodece** [comment]
Pause migration to v2.

**Pull request closed without merging**